tests/test_client.py

```diff
@@ -745,7 +747,7 @@ def update_payment(self, obj_id: str, action=None):
             raise RuntimeError("Require non-blank obj_id, found {}".format(obj_id))
 
         if not action:
-            action = random.choice([ 'complete', 'cancel' ])
+            action = random.choice([ 'complete'])
```

Comment:
Did we determine that the status does indicate the presence of `processing_fee`? If that's the case I think it would be better if we could build that logic into `get_a_payment` rather than remove the path here.

Reply:
I have a cleanup branch for the tests right now so this could be done there if our objective right now is to just get tests green.

Reply:
I put it back, the test should work with either one now.